### PR TITLE
properly handle "" no_proxy option in interactive mode

### PR DESCRIPTION
**Pull request by zvikorn**
…ractive mode

Signed-off-by: Tzvika Avni <tavni@redhat.com>

[SDA-6587](https://issues.redhat.com//browse/SDA-6587)

@zgalor @oriAdler @pvasant  @tbrisker @samira-barouti

### CODE REVIEW

File: cmd/edit/cluster/cmd.go [hunk 1]
```diff
@@ -397,6 +397,10 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	if len(noProxySlice) > 0 {
+		if len(noProxySlice) == 1 && noProxySlice[0] == doubleQuotesToRemove {
+			noProxySlice[0] = ""
+		}
+
```
Comment (tbrisker) [inline cmd/edit/cluster/cmd.go:403]: shouldn't we treat `""` the same as an empty slice?

**Review by tbrisker — COMMENTED** [review]
nitpick - could you change the commit message to describe what changed (e.g. `properly handle "" no_proxy option in interactive mode`) rather than `fix - bug description`?

File: cmd/edit/cluster/cmd.go (hunk 1 above)
Comment (zvikorn) [reply cmd/edit/cluster/cmd.go:403]: no. we need to explicitly treat "" to remove the current value.

Comment (tbrisker) [reply cmd/edit/cluster/cmd.go:403]: Ah, good point.

**Pull request merged**